Assets/MixedRealityToolkit.SDK/Inspectors/UX/Pointers/LinePointerInspector.cs

```diff
@@ -42,12 +43,19 @@ public override void OnInspectorGUI()
             if (linePointerFoldout)
             {
                 EditorGUI.indentLevel++;
+
+                int lineCastResolutionValue = lineCastResolution.intValue;
+                if (lineCastResolutionValue > maxRecommendedLinecastResolution)
+                {
+                    EditorGUILayout.LabelField("Note: values above " + maxRecommendedLinecastResolution + " should only be used when your line is expected to be highly non-uniform.", EditorStyles.miniLabel);
```

Comment:
can you define "highly non-uniform"?

Reply:
A controller styled after a rope, whip or tentacle. Something with lots of bends that needs to 'thread the needle' when detecting collisions.